bakerytest/bakerytest.go

```diff
@@ -221,8 +222,9 @@ func (d *InteractiveDischarger) checker(req *http.Request, cavId, cav string) ([
 	d.id++
 	d.waiting[id] = discharge{cavId, make(chan dischargeResult, 1)}
 	d.mu.Unlock()
-	visitURL := fmt.Sprintf("%s/visit?waitid=%s", d.Discharger.server.URL, id)
-	waitURL := fmt.Sprintf("%s/wait?waitid=%s", d.Discharger.server.URL, id)
+	prefix := strings.TrimSuffix(req.URL.String(), "/discharge")
```

Comment:
The title of the form is derived from the host part of the URL. I have made this change to make the host part of the URL predictable.